serving/samples/telemetry-go/README.md

```diff
@@ -2,100 +2,137 @@
 
 This sample runs a simple web server that makes calls to other in-cluster services
 and responds to requests with "Hello World!".
-The purpose of this sample is to show generating metrics, logs and distributed traces
-(see [Logs](../../accessing-logs.md), [Metrics](../../accessing-metrics.md), and [Traces](../../accessing-traces.md) for more information).
-This sample also creates a dedicated Prometheus instances rather than using the one
-that is installed by default as a showcase of installing dedicated Prometheus instances.
+The purpose of this sample is to show generating [metrics](../../accessing-metrics.md), [logs](../../accessing-logs.md) and distributed [traces](../../accessing-traces.md).
+This sample also shows how to create a dedicated Prometheus instance rather than
+using the default installation.
 
 ## Prerequisites
 
-1. [Install Knative Serving](https://github.com/knative/docs/blob/master/install/README.md)
-2. [Install Knative monitoring component](../../installing-logging-metrics-traces.md)
-3. Install [docker](https://www.docker.com/)
-
+1. A Kubernetes cluster with [Knative Serving](https://github.com/knative/docs/blob/master/install/README.md) installed.
+2. Check if Knative monitoring components are installed:
+```
+kubectl get pods -n monitoring
+```
+  * If pods aren't found, install [Knative monitoring component](../../installing-logging-metrics-traces.md).
+3. Install [Docker](https://docs.docker.com/get-started/#prepare-your-docker-environment).
+4. Check out the code:
+```
+go get -d github.com/knative/docs/serving/samples/telemetry-go
+```
 
 ## Setup
 
-Build the app container and publish it to your registry of choice:
+Build the application container and publish it to a container registry:
```

Comment:
FWIW, these docker build instructions are different than what we use elsewhere.

Perhaps a `<!-- TODO: normalize these with other docs -->` ?

Reply:
I have started to migrate all samples to use this format, in order to be more clear on each step.